sdk/loadtestservice/load-testing-rest/package.json

```
@@ -79,51 +70,61 @@
   "sideEffects": false,
   "autoPublish": false,
   "dependencies": {
-    "@azure/core-auth": "^1.3.0",
     "@azure-rest/core-client": "1.0.0-beta.10",
+    "@azure/core-auth": "^1.3.0",
+    "@azure/core-paging": "^1.2.0",
     "@azure/core-rest-pipeline": "^1.8.0",
     "@azure/logger": "^1.0.0",
     "tslib": "^2.2.0",
-    "@azure/core-paging": "^1.2.0",
-    "@azure/core-util": "1.1.1"
+    "@azure/core-lro": "~2.4.1",
+    "@azure/core-client": "1.6.1"
```

Is @azure/core-client needed?

```suggestion
    "@azure/core-lro": "^2.4.1",
```

---

@azure/core-client is needed to import the { OperationOptions } in creating LRO interface in model.ts, took reference from [https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/notificationhubs/notification-hubs/src/notificationHubsClient.ts](url) .

---

I left a comment that `OperationOptions` is not needed. So please remove this dependency.